lib/irb/command/copy.rb

```diff
@@ -39,8 +39,12 @@ def execute(arg)
       private
 
       def copy_to_clipboard(text)
-        IO.popen(clipboard_program, 'w') do |io|
-          io.write(text)
+        if Gem.win_platform?
```

Comment:
There's a flag `--disable-gems`
```
$ ruby --disable-gems -e binding.irb      
irb(main):001> Gem
uninitialized constant Gem (NameError)
```

I think it's better to add a guard like `defined?(Gem) &&`